ethpm/assets/package.schema.v2.json

```diff
@@ -2,14 +2,14 @@
   "title": "Release Package Specification",
   "type": "object",
   "required": [
-    "spec_version",
+    "manifest_version",
     "package_name",
     "version"
   ],
   "version": "2",
   "additionalProperties": false,
   "properties": {
-    "spec_version": {
+    "manifest_version": {
```

Comment:
I went with "manifest_version" - seemed like a better descriptor and that's what the guys at truffle went with. https://github.com/trufflesuite/ethpm-js/blob/master/templates/epm.json